config/config.php

```diff
@@ -23,10 +23,10 @@
     //   - `*.global.php`
     //   - `local.php`
     //   - `*.local.php`
-    new PhpFileProvider('config/autoload/{{,*.}global,{,*.}local}.php'),
+    new PhpFileProvider(realpath(__DIR__) . '/autoload/{{,*.}global,{,*.}local}.php'),
```

Comment:
Adding this breaks in a docker image on windows. Any other solution that would work on all systems?

Reply:
Actually it does work. I had to remove `config` from the path.

Reply:
Good to hear!